logic/broadcaster.js

```diff
@@ -137,7 +126,6 @@ class Broadcaster {
 	 */
 	broadcast(params, options, cb) {
 		params.limit = params.limit || this.config.peerLimit;
```

Comment:
Are you sure that default souldn't be `self.config.broadcastLimit` instead of `this.config.peerLimit`?

Reply:
Yes I think that would be better, also the logic below can be simplified it's slicing the array multiple times.

Reply:
@jondubois Are you really sure that you want to change behavior? Notice that we are calling `broadcast` here:
https://github.com/LiskHQ/lisk/blob/912d22381a4c3c17447ac4c57b2f00dffbeaac11/modules/transport.js#L438-L442
So before the changes second truncate (to `self.config.broadcastLimit`) was performed, now - it's not. Also `self.config.broadcastLimit` is not used anymore.